designer/server/src/views/file/download-complete.njk

```diff
@@ -0,0 +1,28 @@
+{% extends 'layouts/page.njk' %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block head %}
+  {{ super()}}
+  <meta http-equiv="refresh" content="0; url={{downloadUrl}}">
+{% endblock %}
+
+{% block content %}
+  {% call appPageBody({
+    heading: pageHeading
+  }) %}
+    <p class="govuk-body-l">
+      Your file is downloading
+    </p>
+
+    <p class="govuk-body-l">
+      Once download is complete, your file will be saved in the 'Downloads' folder on your computer. If nothing happens, select 'Download file'.
```

Comment:
```suggestion
      Once download is complete, your file will be saved in the ‘Downloads’ folder on your computer. If nothing happens, select ‘Download file’.
```

Reply:
Should probably use the actual button text too

```suggestion
      Once download is complete, your file will be saved in the ‘Downloads’ folder on your computer. If nothing happens, select ‘{{ buttonText }}’.
```

Reply:
this page will most likely change but I have done above